src/Acts/CamdramBundle/Service/EmailDispatcher.php

```diff
@@ -44,16 +44,29 @@ public function sendShowCreatedEmail(Show $show, array $owners, array $moderator
         $message = \Swift_Message::newInstance()
             ->setSubject('New show needs authorization on Camdram: '.$show->getName())
             ->setFrom(array($this->from_address => 'camdram.net'))
+            
+            /* HTML */
             ->setTo($toEmails)
             ->setBcc($bccEmails)
             ->setBody(
                 $this->twig->render(
+                    'ActsCamdramBundle:Email:show_created.html.twig',
+                    array(
+                        'owners' => $owners,
+                        'show' => $show,
+                    )
+                ), 'text/html'
+            )
+            
+            /* Plain Text */
+            ->addPart(
+                $this->twig->render(
```

Comment:
Try changing `$this->once()` to `$this->exactly(2)` or `$this->atLeastOnce()` in https://github.com/camdram/camdram/blob/master/src/Acts/CamdramBundle/Tests/Service/EmailDispatcherTest.php#L55
Relevant bit of PHPUnit docs: https://phpunit.de/manual/current/en/test-doubles.html#test-doubles.mock-objects.tables.matchers